### ISSUE TITLE
Range slider jumps around on mobile

### ISSUE THREAD

**Issue by stubean** [react-component/slider#603]
On iPhone(version 13.2) and Android(version 10), when you quickly move the slider it will sometimes snap the just moved marker to a previous position. In the gif below, I'm on the example site for this component and I move the marker to the 14% step and it will snap back to 13%.

![image](https://user-images.githubusercontent.com/29029825/68490076-98362a00-0205-11ea-99fa-6a881d644ab3.png)

**Comment by lyxverycool** [comment]
Yes, I also encountered the same problem. If pressing force is more likely to happen, does anyone have a solution?

**Comment by michaelcsilva** [comment]
I see this behavior too, mostly on iOS.

**Comment by scarletks1214** [comment]
On iOS, the slider is not working correctly. (handler is jumping to another point of value)
I tried a react-range library and it goes the same.
Please provide me solutions or any work around...

**Comment by IsaacCalligeros** [comment]
I've been having this issue for the past day. Did anyone find a solution to this?

**Comment by L-Chris** [comment]
Any work around?

**Comment by IsaacCalligeros** [comment]
> Any work around?

I couldn't get anything working with this library or many others. I ended up going with https://github.com/mmarkelov/react-nouislider
I ran into some issues with the noui-slider call backs being a bit slower to render changes than other libraries and had to debounce them but other than that it worked well.

**Comment by L-Chris** [comment]
> > Any work around?
> 
> I couldn't get anything working with this library or many others. I ended up going with https://github.com/mmarkelov/react-nouislider
> I ran into some issues with the noui-slider call backs being a bit slower to render changes than other libraries and had to debounce them but other than that it worked well.

I solved by removing mousedown event from source code. I find it triggers a unnessary mousedown event that caused to this issue.
`
/src/common/createSlider.tsx 340 line
`

**Comment by shivamragnar** [comment]
This is a really weird issue, any updates on this ?

**Comment by Enteleform** [comment]
Same issue here.

**Comment by ninjarogue** [comment]
Any updates... ?

**Comment by dasl-** [comment]
Today I tried using version 10.0.0-alpha.5. I noticed that this bug appears to be fixed in that version. I am not sure which is the first version where this bug was fixed. You can `package.json` like so: `"rc-slider": "10.0.0-alpha.5",`
